app/javascript/mastodon/components/media_gallery.js

```diff
@@ -263,9 +264,9 @@ export default class MediaGallery extends React.PureComponent {
       const size = media.take(4).size;
 
       if (this.isStandaloneEligible()) {
-        children = <Item standalone onClick={this.handleClick} attachment={media.get(0)} />;
+        children = <Item standalone onClick={this.handleClick} attachment={media.get(0)} displayWidth={width} />;
```

Comment:
It appears to work, but I'm not sure how, as `width` is undefined when it's rendered for the first time.

Reply:
Well, not sure what we can do there. Hardcode a fallback width? That's back to status quo with ~330px

Reply:
It's kind of redundant with the CSS, but we can always know which size it is going to be, from the context. That would be best. But apparently it works as is, so… :man_shrugging: